doc/WebSite/Localization.md

```diff
@@ -437,50 +437,50 @@ sets the source name:
         }
     }
 
-And set this view base class in web.config:
+Then set this view base class in web.config:
 
     <pages pageBaseType="SimpleTaskSystem.Web.Views.SimpleTaskSystemWebViewPageBase">
 
-All these for controllers and views are ready when you create your
+All controllers and views are ready with these methods when you create your
 solution from one of the ASP.NET Boilerplate [templates](/Templates).
 
 #### In Javascript
 
-ASP.NET Boilerplate makes possible to use same localization texts in
-also javascript code. First, you should be added dynamic ABP scripts to
+ASP.NET Boilerplate also makes it possible to use the same localization text in
+javascript. First, you need to add the dynamic ABP scripts to
```

Comment:
all the instances of lower-case javascript should be **JavaScript**. Like in the Microsoft doc https://docs.microsoft.com/en-us/scripting/javascript/reference/javascript-reference